### DISCUSSION TITLE
Unable to load the service index for source.

### DISCUSSION THREAD

**Discussion by jonathanromanovmoore** [dotnet/roslyn#43297] [General]
**Version Used**: 
roslyn-Visual-Studio-2017-Version-15.9

**Steps to Reproduce**:

1. Open Admin CMD Window
2. set powershell PATH  
3. run restore.cmd

**Expected Behavior**:
restore complete
**Actual Behavior**:
Unable to load the service index for source https://vside.myget.org/F/vssdk/api v3/index.json.

**Comment by jmarolf** [comment]
@jdm7dv is this against the latest commit in master? That nuget package source is not used anymore in our [NuGet.config](https://github.com/dotnet/roslyn/blob/master/NuGet.config).
